__mocks__/react-relay.js

```diff
@@ -1,11 +1,14 @@
 // Comes from https://github.com/facebook/relay/issues/161
 // Should be used in tests as a replacement for `react-storybooks-relay-container`
 //
-var Relay = require.requireActual('react-relay')
+var Relay = require.requireActual("react-relay")
 
 module.exports = {
   QL: Relay.QL,
   Mutation: Relay.Mutation,
   Route: Relay.Route,
```

Comment:
These are no longer needed are they?

Reply:
I didn't check if there were any references left but if there aren't, it's safe to remove. I'll check now